src/App.js

```diff
@@ -17,8 +18,27 @@ const tabs = [
   },
 ];
 
-const App = () => (
-  <h1>React tabs</h1>
-);
+export class App extends React.Component {
+  state = {
+    titleToPrint: 0,
```

Comment:
Title is not a number. Maybe selectedTabIndex will be a better name for this variable?